ap/aputils/trainee_utils.py

```
@@ -1,13 +1,19 @@
 from accounts.models import Trainee
 
 def is_trainee(user):
-    t = user.type
+    try:
```

All users should have a type, how can this throw? If user doesn't have a type, then is_trainee is being passed the wrong type of object. This can have undesired behavior. If the template does ```{% if not is_TA(user) %} <-- display trainee info (e.g. user.type) --> {% end %}``` this will cause silent errors.